build-tools/android-toolchain/android-toolchain.targets

```diff
@@ -61,7 +61,7 @@
         DestinationFolder="$(AndroidToolchainDirectory)\ndk"
     />
     <Touch
-        Files="$(AndroidToolchainDirectory)\.stamp-sdk"
+        Files="@(_PlatformAndroidSdkItem->'$(AndroidToolchainDirectory)\sdk\%(DestDir)\.stamp-sdk') $(AndroidToolchainDirectory)\ndk\.stamp-ndk"
```

Comment:
This would need to be updated to use the same paths as above, so it might be better to introduce a toplevel `<ItemGroup/>` to list these files to prevent duplication:

    <ItemGroup>
        <_SdkStampFiles Include="@(_PlatformAndroidSdkItem->'$(AndroidToolchainDirectory)\sdk\.stamp-%(Identity)')" />
    </ItemGroup>
    <Target Name="_UnzipFiles"
            ...
            Outputs="@(_SdkStampFiles);$(AndroidToolchainDirectory)\ndk\.stamp-ndk">